src/System.Net.Http/tests/FunctionalTests/HttpCookieProtocolTests.cs

```diff
@@ -151,6 +153,63 @@ private static CookieContainer CreateSingleCookieContainer(Uri uri, string cooki
             });
         }
 
+        [ActiveIssue(31424)]
```

Comment:
@davidsh Since #31424 has been closed, which issue should I use to disable the test, before we come up with an idea regarding HTTP/2 testing?

Reply:
I opened up a new issue #31496 for that.  So, change to that issue number.